heimdall-frontend/source/mainwindow.cpp

```diff
@@ -659,7 +659,7 @@ void MainWindow::SelectPartitionName(int index)
 
 		partitionFileGroup->setTitle(title);
 
-		if (!fileInfo.GetFilename().isEmpty())
+		if (pitEntry && !fileInfo.GetFilename().isEmpty())
```

Comment:
I did not encounter this personally.
Just by looking at the code, you also see a check the pitEntry is not null in line 657.
So if pitEntry can be null there, it also can be null in line 664.
At least that was my reasoning behind at the null check :)